db/get_users_by_emails.sql

```diff
@@ -0,0 +1,3 @@
+SELECT id, email
+FROM users
+WHERE email IN ($1:list)
```

Comment:
I think this is exactly what @KyleD8n and I were looking for! Kyle maybe after all the other cards are done we can refactor to allow users to select multiple categories again? If that's even functionality we still want in the app that is..